docs/samples/deployments/redacting-telemetry/index.md

```
@@ -0,0 +1,106 @@
+## Considerations and Out of Scope
+
+This demo intentionally does NOT cover:
+
+- Production deployment patterns
+- Configurations, NLP model selection, or custom entity recognizers
+- Reliability & Performance optimization
+- Alternative architectures (sidecar, (otel-)collector-side masking, et cetera)
+
+**Architecture:**
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)